image.yaml

```diff
@@ -46,14 +46,10 @@ modules:
             version: '1.0'
 
 artifacts:
-      - md5: 34a8787d5f5cdf54d5c1477ab1bdf47b
-        name: jboss-eap-7.2.7-patch
-        target: jboss-eap-7.2.7-patch.zip
-        url: http://$DOWNLOAD_SERVER/devel/candidates/JBEAP/JBEAP-7.2.7.CP-CR3/jboss-eap-7.2.7.CP-CR3-patch.zip
-      - md5: c8ecfa7a0c68c3c21a3333b1e5a9c343
```

Comment:
@drichtarik The change in overall LGTM (wrt to updating image versions & utilizing the updated EAP-7.2.8.CR1 JAR). 

But unsure if it's OK to drop the `jbeap-18811.zip` artifact below just due the fact we would rebase on top of EAP-7.2.8.CR1. <- @hmlnarik @pskopek Can you suggest / advice, if it's OK to drop it already?

Thanks!

Reply:
Any one-off should be fixed in the next micro release and dropped from the previous image. So this change is correct.

Reply:
@hmlnarik Ok, great, thanks for clarification!